JPS_Ontology/ontology/ontoheatnetwork/ontoheatnetwork.owl

```diff
@@ -120,88 +120,88 @@
+        <rdfs:range rdf:resource="http://theworldavatar.com/ontology/ontoheatnetwork/OntoHeatNetwork.owl#UnitPrice"/>
         <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">A relationship between contract and current unit price based on relative purchases according to tiers</rdfs:comment>
         <rdfs:isDefinedBy rdf:datatype="http://www.w3.org/2001/XMLSchema#string">https://www.theworldavatar.com/kg/ontoheatnetwork/</rdfs:isDefinedBy>
         <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">has current unit price</rdfs:label>
     </owl:ObjectProperty>
     
 
 
-    <!-- https://www.theworldavatar.com/kg/ontoheatnetwork/hasDownstreamGridConnection -->
+    <!-- http://theworldavatar.com/ontology/ontoheatnetwork/OntoHeatNetwork.owl#hasDownstreamGridConnection -->
 
 
-    <owl:ObjectProperty rdf:about="https://www.theworldavatar.com/kg/ontoheatnetwork/hasDownstreamGridConnection">
-        <rdfs:domain rdf:resource="https://www.theworldavatar.com/kg/ontoheatnetwork/HeatProvider"/>
-        <rdfs:range rdf:resource="https://www.theworldavatar.com/kg/ontoheatnetwork/GridConnection"/>
+    <owl:ObjectProperty rdf:about="http://theworldavatar.com/ontology/ontoheatnetwork/OntoHeatNetwork.owl#hasDownstreamGridConnection">
+        <rdfs:domain rdf:resource="http://theworldavatar.com/ontology/ontoheatnetwork/OntoHeatNetwork.owl#HeatProvider"/>
+        <rdfs:range rdf:resource="http://theworldavatar.com/ontology/ontoheatnetwork/OntoHeatNetwork.owl#GridConnection"/>
         <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">A relationship between heat provider and downstream grid connection, i.e., flow</rdfs:comment>
         <rdfs:isDefinedBy rdf:datatype="http://www.w3.org/2001/XMLSchema#string">https://www.theworldavatar.com/kg/ontoheatnetwork/</rdfs:isDefinedBy>
         <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">has downstream grid connection</rdfs:label>
     </owl:ObjectProperty>
     
 
 
-    <!-- https://www.theworldavatar.com/kg/ontoheatnetwork/hasGeneratedHeatAmount -->
+    <!-- http://theworldavatar.com/ontology/ontoheatnetwork/OntoHeatNetwork.owl#hasGeneratedHeatAmount -->
 
 
-    <owl:ObjectProperty rdf:about="https://www.theworldavatar.com/kg/ontoheatnetwork/hasGeneratedHeatAmount">
-        <rdfs:domain rdf:resource="https://www.theworldavatar.com/kg/ontoheatnetwork/HeatGenerator"/>
-        <rdfs:range rdf:resource="https://www.theworldavatar.com/kg/ontoheatnetwork/EnergyInTimeInterval"/>
+    <owl:ObjectProperty rdf:about="http://theworldavatar.com/ontology/ontoheatnetwork/OntoHeatNetwork.owl#hasGeneratedHeatAmount">
+        <rdfs:domain rdf:resource="http://theworldavatar.com/ontology/ontoheatnetwork/OntoHeatNetwork.owl#HeatGenerator"/>
+        <rdfs:range rdf:resource="http://theworldavatar.com/ontology/ontoheatnetwork/OntoHeatNetwork.owl#EnergyInTimeInterval"/>
```

Comment:
I believe the definition of all subClasses of `EnergyInTimeInterval` looks good; however, didn't we want to use those subclasses to refine the TBox for the relevant relationships? i.e. `hasGeneratedHeatAmount` to link to a `GeneratedHeatAmount` concept instead of the general `EnergyInTimeInterval`?
![image](https://user-images.githubusercontent.com/79832739/202326633-955b1c58-ac0f-44ea-a112-79ac42924c79.png)

Reply:
Addressed by [fcf5791](https://github.com/cambridge-cares/TheWorldAvatar/pull/453/commits/fcf57911c5302e0665f23a1556bcf3621cca903b)